packages/newspringchurchapp/src/prayer/AddPrayer/AddPrayerCard/AddPrayerCard.js

```diff
@@ -17,19 +16,18 @@ const StyledView = styled(({ theme }) => ({
   paddingVertical: theme.sizing.baseUnit * 4,
   borderBottomLeftRadius: theme.sizing.baseUnit,
   backgroundColor: theme.colors.paper,
-}))(FlexedView);
+  justifyContent: 'center',
+}))(View);
 
 const StyledBodyText = styled(({ theme }) => ({
   paddingVertical: theme.sizing.baseUnit,
 }))(BodyText);
 
-const StyledCardContent = styled({
-  flex: 1,
-})(CardContent);
+const StyledCardContent = styled({})(CardContent);
```

Comment:
```suggestion
```

Reply:
@redreceipt What's the point of the empty object here? Are you just removing the `flex`? If so, we could just remove `StyledCardContent` completely and just go with `CardContent`, couldn't we?

Reply:
Oh wait ... nevermind. I see now. You are removing `StyledCardContent`. I got it. Just took me a minute. ;)